generators/client/templates/vue/package.json.ejs

```diff
@@ -14,6 +14,7 @@
     "axios": "^0.18.0",
     "bootstrap": "^4.1.3",
     "bootstrap-vue": "^2.0.0-rc.11",
+    "date-fns": "^2.0.0-alpha.26",
```

Comment:
I would go with the upcoming 2.x version as there are some breaking changes from 1.x -> 2.x